dicom_utils/dicom.py

```diff
@@ -280,15 +281,21 @@ def read_dicom_image(
 
     # apply volume handling for 3D data
     if len(dims) == 4:
+        if not isinstance(volume_handler, VolumeHandler):
```

Comment:
Could this code just be rewritten as this?
```python
        if not isinstance(volume_handler, ReduceVolume):
            raise TypeError(f"volume_handler must be a ReduceVolume, not {type(volume_handler)}")
        dcm = volume_handler(dcm, use_nvjpeg, batch_size=nvjpeg_batch_size)
```

Reply:
We still want to support other `VolumeHandler` types, there's no need to constrain specifically to `ReduceVolume`